src/Microsoft.SqlTools.ServiceLayer/SqlProjects/SqlProjectsService.cs

```diff
@@ -117,7 +118,14 @@ internal async Task HandleCreateSqlProjectRequest(Contracts.CreateSqlProjectPara
         {
             await RunWithErrorHandling(async () =>
             {
-                await SqlProject.CreateProjectAsync(requestParams.ProjectUri, new SqlServer.Dac.Projects.CreateSqlProjectParams() { ProjectType = requestParams.SqlProjectType, DspVersion = requestParams.DatabaseSchemaProvider });
+                SqlServer.Dac.Projects.CreateSqlProjectParams createParams = new()
+                {
+                    ProjectType = requestParams.SqlProjectType,
+                    TargetPlatform = requestParams.DatabaseSchemaProvider == null ? null : DspToSqlPlatform(requestParams.DatabaseSchemaProvider),
+                    BuildSdkVersion = requestParams.BuildSdkVersion
```

Comment:
Found that BuildSdkVersion wasn't getting carried over.  Whoops.

Reply:
oh wow good catch